pyproject.toml

```diff
@@ -1,19 +1,25 @@
 [project]
```

Comment:
I think you'll have to remove this

Reply:
Oops my bad, will make the change. I always do `poetry check` and this went through it and didn't complain.